images/mint.yaml

```diff
@@ -407,6 +407,17 @@ actions:
     getent group sudo >/dev/null 2>&1 || groupadd --system sudo
     useradd --create-home -s /bin/bash -G sudo -U ubuntu
 
+- trigger: post-packages
+  action: |-
+    #!/bin/sh
+    set -eux
+
+    # Enable systemd-networkd
```

Comment:
Any reason not to just run `systemctl enable systemd-networkd`?
Because based on my local tests, you're missing a bunch of links here.

You also need this restricted to those releases that use netplan but not restricted to cloud.
I don't know by what magic things are working for non-cloud right now but we may as well be consistent.

Reply:
Should be fine now